Italian/guardare/guarda.md

```diff
@@ -3,7 +3,7 @@ i funghi "speciali" che ho mangiato ieri in realtà erano allucinogeni.
 
 Non avresti dovuto fidarti di quella ragazza conosciuta alla festa.
 
-Hai pochi, confusi ricordi di quello che è successo. Con la test che gira per l'intossicazione, provi ad alzarti.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.